corehq/apps/appstore/views.py

```diff
@@ -286,6 +286,9 @@ def approve_app(request, snapshot):
 @login_required
 @retry_resource(3)
 def import_app(request, snapshot):
```

Comment:
Since this function doesn't create a new project space, I don't think we need to assign a community subscription here.